modules/videosipgw/JitsiVideoSIPGWSession.js

```diff
@@ -140,20 +140,13 @@ export default class JitsiVideoSIPGWSession extends Listenable {
             .c('jibri', attributes)
             .up();
 
-        logger.log('Stop video SIP GW session', iq.nodeTree);
+        logger.debug('${action} video SIP GW session', iq.nodeTree);
         this.chatRoom.connection.sendIQ(
             iq,
-            result => {
-                logger.log('Result', result);
-                const initialState
-                    = $(result).find('jibri')
-                        .attr('state');
-
-                this.setState(initialState);
-            },
+            () => {}, // eslint-disable-line no-empty-function
```

Comment:
It's okay to remove any success handling?

Reply:
Yep. There is nothing to handle. Everything is signaled through events(iq messages) after that.